<ui version="4.0" >
 <class>UIYabause</class>
 <widget class="QMainWindow" name="UIYabause" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Yabause Qt GUI</string>
  </property>
  <widget class="QWidget" name="centralwidget" />
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>320</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="mYabause" >
    <property name="title" >
     <string>&amp;Yabause</string>
    </property>
    <widget class="QMenu" name="mYabauseSaveState" >
     <property name="title" >
      <string>Save State</string>
     </property>
     <addaction name="aYabauseSaveStateSave" />
     <addaction name="aYabauseSaveStateLoad" />
    </widget>
    <addaction name="aYabauseSettings" />
    <addaction name="aYabauseRun" />
    <addaction name="aYabausePause" />
    <addaction name="aYabauseReset" />
    <addaction name="aYabauseTransfer" />
    <addaction name="aYabauseScreenshot" />
    <addaction name="aYabauseFrameSkipLimiter" />
    <addaction name="mYabauseSaveState" />
    <addaction name="separator" />
    <addaction name="aYabauseQuit" />
    <addaction name="separator" />
   </widget>
   <widget class="QMenu" name="mView" >
    <property name="title" >
     <string>&amp;View</string>
    </property>
    <widget class="QMenu" name="mViewLayer" >
     <property name="title" >
      <string>&amp;Layer</string>
     </property>
     <addaction name="aViewLayerVdp1" />
     <addaction name="aViewLayerNBG0" />
     <addaction name="aViewLayerNBG1" />
     <addaction name="aViewLayerNBG2" />
     <addaction name="aViewLayerNBG3" />
     <addaction name="aViewLayerRBG1" />
    </widget>
    <addaction name="aViewFPS" />
    <addaction name="mViewLayer" />
    <addaction name="aViewFullscreen" />
    <addaction name="aViewLog" />
   </widget>
   <widget class="QMenu" name="mViewDebug" >
    <property name="title" >
     <string>&amp;Debug</string>
    </property>
    <addaction name="aViewDebugMSH2" />
    <addaction name="aViewDebugSSH2" />
    <addaction name="aViewDebugVdp1" />
    <addaction name="aViewDebugVdp2" />
    <addaction name="aViewDebugM68K" />
    <addaction name="aViewDebugSCUDSP" />
    <addaction name="aViewDebugSCSP" />
    <addaction name="separator" />
    <addaction name="aViewDebugMemoryDump" />
   </widget>
   <widget class="QMenu" name="mHelp" >
    <property name="title" >
     <string>&amp;Help</string>
    </property>
    <addaction name="aHelpAbout" />
   </widget>
   <addaction name="mYabause" />
   <addaction name="mView" />
   <addaction name="mViewDebug" />
   <addaction name="mHelp" />
  </widget>
  <action name="aYabauseQuit" >
   <property name="text" >
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="aYabauseStart" >
   <property name="text" >
    <string>Start</string>
   </property>
  </action>
  <action name="aYabauseStop" >
   <property name="text" >
    <string>Stop</string>
   </property>
  </action>
  <action name="aYabauseSettings" >
   <property name="text" >
    <string>&amp;Settings...</string>
   </property>
  </action>
  <action name="aYabauseRun" >
   <property name="text" >
    <string>R&amp;un</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="aYabausePause" >
   <property name="text" >
    <string>&amp;Pause</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="aYabauseReset" >
   <property name="text" >
    <string>&amp;Reset</string>
   </property>
  </action>
  <action name="aYabauseTransfer" >
   <property name="text" >
    <string>Transfer</string>
   </property>
  </action>
  <action name="aYabauseScreenshot" >
   <property name="text" >
    <string>Screenshot</string>
   </property>
  </action>
  <action name="aYabauseFrameSkipLimiter" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>Frame Skip/Limiter</string>
   </property>
  </action>
  <action name="aYabauseSaveStateSave" >
   <property name="text" >
    <string>Save</string>
   </property>
  </action>
  <action name="aYabauseSaveStateLoad" >
   <property name="text" >
    <string>Load</string>
   </property>
  </action>
  <action name="aViewFPS" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>FPS</string>
   </property>
  </action>
  <action name="aViewLayerVdp1" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="checked" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>Vdp1</string>
   </property>
  </action>
  <action name="aViewFullscreen" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>&amp;Fullscreen</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="aViewLog" >
   <property name="text" >
    <string>Log</string>
   </property>
  </action>
  <action name="aViewDebugMSH2" >
   <property name="text" >
    <string>MSH2</string>
   </property>
  </action>
  <action name="aViewDebugSSH2" >
   <property name="text" >
    <string>SSH2</string>
   </property>
  </action>
  <action name="aViewDebugVdp1" >
   <property name="text" >
    <string>Vdp1</string>
   </property>
  </action>
  <action name="aViewDebugVdp2" >
   <property name="text" >
    <string>Vdp2</string>
   </property>
  </action>
  <action name="aViewDebugM68K" >
   <property name="text" >
    <string>M68K</string>
   </property>
  </action>
  <action name="aViewDebugSCUDSP" >
   <property name="text" >
    <string>SCU-DSP</string>
   </property>
  </action>
  <action name="aViewDebugSCSP" >
   <property name="text" >
    <string>SCSP</string>
   </property>
  </action>
  <action name="aViewDebugMemoryDump" >
   <property name="text" >
    <string>Memory Dump</string>
   </property>
  </action>
  <action name="aHelpAbout" >
   <property name="text" >
    <string>&amp;About...</string>
   </property>
  </action>
  <action name="aViewLayerNBG0" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="checked" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>NBG0</string>
   </property>
  </action>
  <action name="aViewLayerNBG1" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="checked" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>NBG1</string>
   </property>
  </action>
  <action name="aViewLayerNBG2" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="checked" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>NBG2</string>
   </property>
  </action>
  <action name="aViewLayerNBG3" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="checked" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>NBG3</string>
   </property>
  </action>
  <action name="aViewLayerRBG1" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="checked" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>RBG1</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>aYabauseQuit</sender>
   <signal>triggered()</signal>
   <receiver>UIYabause</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>159</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
